src/Draco.Compiler/Internal/Symbols/Metadata/MetadataTypeSymbol.cs

```diff
@@ -31,6 +33,9 @@ internal sealed class MetadataTypeSymbol : TypeSymbol, IMetadataSymbol
 
     public MetadataReader MetadataReader => this.Assembly.MetadataReader;
 
+    public string? DefaultMemberAttributeName => this.defaultName ??= MetadataSymbol.GetDefaultMemberAttributeName(this.typeDefinition, this.DeclaringCompilation!, this.MetadataReader);
+    private string? defaultName;
```

Comment:
Why not call it `defaultMemberAttributeName`? Naming the prop and cache fields differently is confusing